sdk/storage/azure-storage-files-datalake/vcpkg/vcpkg.json

```diff
@@ -15,7 +15,7 @@
     {
       "name": "azure-storage-blobs-cpp",
       "default-features": false,
-      "version>=": "12.0.0"
+      "version>=": "12.1.0"
```

Comment:
We may need to get of the version constraints for a while, until vcpkg fixes the underlying issue - https://github.com/microsoft/vcpkg/pull/19299#issuecomment-894497720 (there is no reply so far, but I talked to them privately, and we are understanding the version constraints correctly and were using them as intended, but since there is a corner case in one of the vcpkg scenarios, we'll refrain from using them for now, until we are notified to re-enable the,).

I'll create a PR for that soon, once I'm done experimenting with what we can leave.

Reply:
What should I do for Aug. release? Should I just remove this line of code, or just revert the change?

Reply:
@Jinming-Hu, in #2718, I am removing `version>=` and `builtin-baseline` - we can't use them for a while, unfortunately. Not our fault.

Reply:
@antkmsft Sure, you merge #2718 first, I'll resolve the conflicts.
You'll merge #2718 before Aug. 10, right?

Reply:
Yes, it is merged now.